.husky/.secret.patterns

```diff
@@ -0,0 +1,6 @@
+smil\.mil
+(us|aws)-iso
+[aA]pollo|[bB]razil|[cC]oral|[oO]din
+tt\.amazon\.com|issues\.amazon\.com|cr\.amazon\.com
+ic\.gov|sgov\.gov
+(us|aws)-iso
```

Comment:
Is this file needed? Some of these are internal and we shouldn't push in a public repo imo.

Reply:
It's needed to check for those patterns, so if we don't need to worry about those then we could omit it. If we still want to check for those we could make adding this file a manual configuration step (but that sort of defeats the purpose). 
 
For the hook to look for those patterns without a configuration step they need to be somewhere, maybe if there's a secure place we could host it somewhere? Idk if these particular patterns are a different case but other patterns are listed on the awslabs public repo though https://github.com/awslabs/git-secrets#options-for-add-provider

Reply:
I would assume we wouldn't have to deal with lines 3 and 4 in any way?